### PR TITLE
Move Ibft event exeuctors to common

### PR DESCRIPTION
**Pull request by rain-on**
Move more classes to common such that they can be reused for QBFT.

### CODE REVIEW

File: consensus/common/src/main/java/org/hyperledger/besu/consensus/common/bft/IbftExecutors.java [hunk 1]
```diff
@@ -61,7 +60,8 @@ public synchronized void start() {
     }
     state = State.RUNNING;
     ibftProcessorExecutor = Executors.newSingleThreadExecutor();
-    timerExecutor = newScheduledThreadPool("IbftTimerExecutor", 1, metricsSystem);
+    timerExecutor =
+        MonitoredExecutors.newScheduledThreadPool("IbftTimerExecutor", 1, metricsSystem);
```
Comment (jframe) [inline consensus/common/src/main/java/org/hyperledger/besu/consensus/common/bft/IbftExecutors.java:64]: Thread pool should probably be configurable or perhaps just make it "BftTimerExecutor"

File: consensus/common/src/main/java/org/hyperledger/besu/consensus/common/bft/IbftEventQueue.java [hunk 2]
```diff
@@ -12,7 +12,7 @@
  *
  * SPDX-License-Identifier: Apache-2.0
  */
-package org.hyperledger.besu.consensus.ibft;
+package org.hyperledger.besu.consensus.common.bft;
 
 import org.hyperledger.besu.consensus.common.bft.events.IbftEvent;
```
Comment (jframe) [inline consensus/common/src/main/java/org/hyperledger/besu/consensus/common/bft/IbftEventQueue.java:18]: Suggest renamed to BftEventQueue

File: consensus/common/src/main/java/org/hyperledger/besu/consensus/common/bft/IbftExecutors.java [hunk 3]
```diff
@@ -13,10 +13,9 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-package org.hyperledger.besu.consensus.ibft;
-
-import static org.hyperledger.besu.ethereum.eth.manager.MonitoredExecutors.newScheduledThreadPool;
+package org.hyperledger.besu.consensus.common.bft;
 
+import org.hyperledger.besu.ethereum.eth.manager.MonitoredExecutors;
```
Comment (jframe) [inline consensus/common/src/main/java/org/hyperledger/besu/consensus/common/bft/IbftExecutors.java:18]: Suggest rename of class to BftExecutors

File: consensus/common/src/main/java/org/hyperledger/besu/consensus/common/bft/IbftProcessor.java [hunk 4]
```diff
@@ -12,9 +12,8 @@
  *
  * SPDX-License-Identifier: Apache-2.0
  */
-package org.hyperledger.besu.consensus.ibft;
+package org.hyperledger.besu.consensus.common.bft;
```
Comment (jframe) [inline consensus/common/src/main/java/org/hyperledger/besu/consensus/common/bft/IbftProcessor.java:16]: Suggest rename of class to BftProcessor

File: consensus/common/src/main/java/org/hyperledger/besu/consensus/common/bft/IbftExecutors.java (hunk 1 above)
Comment (rain-on) [reply consensus/common/src/main/java/org/hyperledger/besu/consensus/common/bft/IbftExecutors.java:64]: renamed

File: consensus/common/src/main/java/org/hyperledger/besu/consensus/common/bft/IbftExecutors.java (hunk 3 above)
Comment (rain-on) [reply consensus/common/src/main/java/org/hyperledger/besu/consensus/common/bft/IbftExecutors.java:18]: done

File: consensus/common/src/main/java/org/hyperledger/besu/consensus/common/bft/IbftProcessor.java (hunk 4 above)
Comment (rain-on) [reply consensus/common/src/main/java/org/hyperledger/besu/consensus/common/bft/IbftProcessor.java:16]: done

File: consensus/common/src/main/java/org/hyperledger/besu/consensus/common/bft/IbftEventQueue.java (hunk 2 above)
Comment (rain-on) [reply consensus/common/src/main/java/org/hyperledger/besu/consensus/common/bft/IbftEventQueue.java]: done

File: consensus/common/src/main/java/org/hyperledger/besu/consensus/common/bft/events/BlockTimerExpiry.java [hunk 5]
```diff
@@ -34,8 +34,8 @@ public BlockTimerExpiry(final ConsensusRoundIdentifier roundIdentifier) {
   }
 
   @Override
-  public IbftEvents.Type getType() {
-    return IbftEvents.Type.BLOCK_TIMER_EXPIRY;
+  public BftEvents.Type getType() {
+    return BftEvents.Type.BLOCK_TIMER_EXPIRY;
   }
 
   public ConsensusRoundIdentifier getRoundIndentifier() {
```
Comment (jframe) [inline consensus/common/src/main/java/org/hyperledger/besu/consensus/common/bft/events/BlockTimerExpiry.java:41]: I know it's not part of your change. But would be nice to fix the misspelling here. /getRoundIndentifier/getRoundIdentifier

**Review by jframe — APPROVED** [review]
LGTM

**Pull request merged**